taxi/admin.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    search_fields = ["country"]
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    search_fields = ["model"]
+    list_filter = ["manufacturer"]
+
+
+@admin.register(Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("licence_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        (
+            "Licence data",
+            {"fields": ("licence_number",)}
+        ),
+    )
+    add_fieldsets = (
+        (
+            None,
+            {
+                "classes": ("wide",),
```

Comment:
Here I am overriding "add_fieldsets" and giving new values instead of the default ones
![image](https://user-images.githubusercontent.com/74211253/177039508-ef999a0e-90d7-4eab-9e30-1a762cf29d4e.png)